libbeat/otelbeat/providers/mbprovider/mbprovider.go

```diff
@@ -0,0 +1,52 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package mbprovider
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/confmap"
+
+	"github.com/elastic/beats/v7/libbeat/otelbeat/providers"
+)
+
+const schemeName = "mb"
+
+type provider struct{}
+
+// The Provider provides configuration, and allows to watch/monitor for changes.
+func NewFactory() confmap.ProviderFactory {
+	return confmap.NewProviderFactory(newProvider)
+}
+
+func newProvider(confmap.ProviderSettings) confmap.Provider {
+	return &provider{}
+}
+
+// Retrieve retrieves the beat configuration file and constructs otel config
+func (fmp *provider) Retrieve(_ context.Context, uri string, _ confmap.WatcherFunc) (*confmap.Retrieved, error) {
+	return providers.LoadConfig(uri, schemeName)
+}
+
+func (*provider) Scheme() string {
+	return schemeName
+}
+
+func (*provider) Shutdown(context.Context) error {
+	return nil
+}
```

Comment:
For the sake of consistency, add the same receiver name and a small documentation about the methods:
```suggestion
// Scheme returns the scheme name
func (p *provider) Scheme() string {
	return schemeName
}

// Shutdown is a noop, it always returns nil
func (p *provider) Shutdown(context.Context) error {
	return nil
}
```